relayer/README.md

```diff
@@ -0,0 +1,59 @@
+# Beamer's cross-chain message relayer
+
+This software component provides a CLI that can be used to relay messages between two networks.
+
+## Setup
+
+Pre-requisites:
+
+- [Node.js >=18.x](https://nodejs.org/en)
+
+Start by installing the dependencies
+
+```
+yarn install
+```
+
+## Compile the TS code
+
+```
+yarn build
+```
+
+## Run
+
+There are two ways to run the CLI commands.
+
+1. By using the TS runtime enviroment `ts-node` (preffered in dev mode)
+
+```
+yarn ts-node ./src/cli.ts <command> [options]
+```
+
+2. By using node after running the [compilation step above](#compile-the-ts-code)
+
+```
+node ./build/src/cli.js <command> [options]
+```
+
+You can find more info on the available commands and their respective options [here](https://docs.beamerbridge.com/relayer).
+
+If you forgot the commands & options, you can always consult yourself via the CLI by running it with `--help`.
```

Comment:
I'd drop the "forgetting" part and reformulate in a neutral tone, e.g
```
Available commands and options can be listed using the `--help` option.
```
or something like that.